packages/ui-components/src/lib/charts/PieChart/PieChart.tsx

```diff
@@ -11,6 +11,44 @@ export interface PieChartProps extends Partial<ChartBaseProps> {
   data?: PieChartData;
 }
 
```

Comment:
I just rearranged the order of the types/functions in this file, since `PieChartLegend` was being used before is was assigned.